prov/psm2/src/makefile

```diff
@@ -0,0 +1,7 @@
+setup:
+	cp ../../psm/src/psmx*.[ch] .
```

Comment:
This seems like a bad idea.  Can you make sym links, instead?  Sym links can be committed to git. You can probably get rid of this "makefile" then...?